### ISSUE TITLE
Regression 0.5.79 -> 0.5.80: CS no brace hash

### ISSUE THREAD

**Issue by greghuc** [DanielXMoore/Civet#434]
### Coffeescript bug (broken code)

Coffeescript code ([Try Coffeescript link](https://coffeescript.org/#try:%22civet%20coffeeCompat%22%0A%0AnewActions%20%3D%20(actionHash)%20-%3E%20actionHash%5B'start'%5D()%0A%0Aactions%20%3D%20newActions%0A%20%20start%3A%20-%3E%20console.log%20'Action%20start'%0A%20%20stop%3A%20-%3E%20console.log%20'Action%20stop'))
```
"civet coffeeCompat"

newActions = (actionHash) -> actionHash['start']()

actions = newActions
  start: -> console.log 'Action start'
  stop: -> console.log 'Action stop'
```
0.5.79 gives working 
```
var newActions, actions
newActions = function(actionHash) { return actionHash['start']() }

actions = newActions({
  start: function() { return console.log('Action start') },
  stop: function() { return console.log('Action stop') },
})
```
0.5.80+ in [Civet playground](https://civet.dev/playground?code=ImNpdmV0IGNvZmZlZUNvbXBhdCIKCm5ld0FjdGlvbnMgPSAoYWN0aW9uSGFzaCkgLT4gYWN0aW9uSGFzaFsnc3RhcnQnXSgpCgphY3Rpb25zID0gbmV3QWN0aW9ucwogIHN0YXJ0OiAtPiBjb25zb2xlLmxvZyAnQWN0aW9uIHN0YXJ0JwogIHN0b3A6IC0%2BIGNvbnNvbGUubG9nICdBY3Rpb24gc3RvcCc%3D) gives broken code
```
"civet coffeeCompat"

newActions = (actionHash) -> actionHash['start']()

actions = newActions
  start: -> console.log 'Action start'
  ^ ParseError
```

**Comment by STRd6** [comment]
My guess is it's related to changes to the function code here: https://github.com/DanielXMoore/Civet/pull/433

**Comment by STRd6** [comment]
> ```
> actions = newActions
>   start: -> console.log 'Action start'
>   stop: -> console.log 'Action stop'
> ```

Actually... looks like it was this commit: https://github.com/DanielXMoore/Civet/pull/431

**Comment by STRd6** [comment]
Found the real one, next time I'll finish the bisect before commenting :P

https://github.com/DanielXMoore/Civet/commit/082fd343b2398ec97925440ca4a1022fd59b5fe3

**Comment by edemaine** [comment]
Fixed in 0.5.83, thanks to @STRd6!

**Comment by greghuc** [comment]
Thanks both!
